hack/invalid-examples/v1alpha2/gateway/duplicate-listeners.yaml

```diff
@@ -1,13 +0,0 @@
-apiVersion: gateway.networking.k8s.io/v1alpha2
```

Comment:
Why are so many of thee being removed?

Reply:
Yes, there are actually more examples in v1beta1, but all the ones being removed are covered already.